lib/masamune/environment.rb

```diff
@@ -151,14 +149,5 @@ def log_file_io
         configuration.debug ? $stderr : nil
       end
     end
-
-    def symlink_latest_log
-      return unless filesystem.has_path?(:log_dir)
-      latest = filesystem.path(:log_dir, 'latest')
-      FileUtils.rm(latest) if File.exists?(latest)
-      FileUtils.ln_s(log_file_name, latest)
-    rescue => e
-      logger.error(e)
-    end
```

Comment:
this is nice to have when executing a single task, but problematic when executing tasks in parallel- removing